Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Dockerfile
+
+FROM python:3.8-alpine
+
+LABEL maintainer="mapachana"
+
+WORKDIR /app
+COPY ./poetry.lock ./pyproject.toml ./tasks.py /app/
+
+RUN apk update; apk add curl; ln -s /usr/bin/python /usr/bin/python3.8; addgroup -S testgroup && adduser -S test -G testgroup
+USER test
+WORKDIR /app/test
+
+RUN pip3 install invoke; wget -q -O - "$@" https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+
+ENV PATH=$PATH:/home/test/.local/bin
+ENV PATH=$PATH:/home/test/.poetry/bin
+
+RUN  poetry config virtualenvs.create false; invoke installdeps --dev
```

Comment:
Sigo sin entender por qué hacéis esto. ¿Qué pasa si lo quitas?

Reply:
Si lo quito poetry instala las dependencias en un entorno virtual propio, no en el contenedor en sí, lo cuál es innecesario porque estamos usando docker justamente para que estas cosas no sean necesarias.
Además, al instalar las dependencias en un entorno virtual no las encuentras después durante la ejecución.
Por eso se desactiva la creación de entornos virtuales